tests/pytests/conftest.py

```diff
@@ -2,6 +2,22 @@
 import pytest
 
 
+def pytest_addoption(parser):
+    parser.addoption(
+        "--runnoci", action="store_true", default=False, help="run the noci tests in stdsuite"
```

Comment:
"noci" turns off most `grd0`, `hes0`, `hes1` testing and all fci method tests. the grd0 (aka G by E findif) can by up to 80% of time consumed, especially when oo involved.

Reply:
Can we do this by making `findif` and `fci` tags?

Reply:
leaving aside fci, one can do this now with `pytest -k "not grd0"` (grd0 is the main culprit since there aren't many Hessian tests). and because I've already added the "noci" marker to the test_standard_suite.py file, `pytest -m "not noci"` does the same thing and doesn't collect any of those.

what the `--runnoci` option that you've highlighted does is turn them off _by default_, while giving the option (w/o toggling comments in code) of turning them back on. this is the pytest endorsed way of turning off subsets of tests by default.

Reply:
I'm fine with this mechanism of having tests off by default. I just would like a more descriptive name for this option. `noci` looks like it's only about FCI, not FCI + FINDIF.

Reply:
`nonroutine` is the best I've come up with. `ci` here I was thinking continuous integration, btw.